src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -20,9 +43,158 @@ public class MauiMediaElement : Grid, IDisposable
 	public MauiMediaElement(MediaPlayerElement mediaPlayerElement)
 	{
 		this.mediaPlayerElement = mediaPlayerElement;
+
+		btn = new Button
+		{
+			Background = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Transparent),
+			Foreground = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Transparent),
+			Width = 40,
+			Height = 40
+		};
+		btn.Click += ButtonSetFullScreenStatus;
+
+		Image image = new()
+		{
+			Width = 40,
+			Height = 40,
+			HorizontalAlignment = Microsoft.UI.Xaml.HorizontalAlignment.Right,
+			VerticalAlignment = Microsoft.UI.Xaml.VerticalAlignment.Top,
+			Source = new BitmapImage(new Uri("ms-appx:///whitefs.png")),
+		};
+
+		ButtonContainer = new()
+		{
+			HorizontalAlignment = Microsoft.UI.Xaml.HorizontalAlignment.Right,
+			VerticalAlignment = Microsoft.UI.Xaml.VerticalAlignment.Top,
+			Width = 40,
+			Height = 40
+		};
+		ButtonContainer.Children.Add(image);
+		ButtonContainer.Children.Add(btn);
+
 		Children.Add(this.mediaPlayerElement);
+		Children.Add(ButtonContainer);
+
+		mediaPlayerElement.PointerMoved += MediaPlayerElement_PointerMoved;
+	}
+
+	void MediaPlayerElement_PointerMoved(object sender, PointerRoutedEventArgs e)
+	{
+		e.Handled = true;
+		mediaPlayerElement.PointerMoved -= MediaPlayerElement_PointerMoved;
+		InitializeTimer();
+		ButtonContainer.Visibility = Microsoft.UI.Xaml.Visibility.Visible;
+	}
+
+	void InitializeTimer()
+	{
+		if (timer is not null)
+		{
+			return;
+		}
+
+		timer = new();
+		timer.Tick += OnTimer_Tick;
```

Comment:
```suggestion
		timer.Tick += OnTimerTick;
```

Reply:
Also, if I understand it correctly, this Ticker jazz is to hide the button after 5 seconds, right? Wouldn't be easier to make `MediaPlayerElement_PointerMoved` async and add a Task.Delay(TimeSpan.FromSeconds(5))?

```cs
async void MediaPlayerElement_PointerMoved(object sender, PointerRoutedEventArgs e)
	{
		e.Handled = true;
		mediaPlayerElement.PointerMoved -= MediaPlayerElement_PointerMoved;
		ButtonContainer.Visibility = Microsoft.UI.Xaml.Visibility.Visible;
		await Task.Delay(TimeSpan.FromSeconds(5));
		ButtonContainer.Visibility = Microsoft.UI.Xaml.Visibility.Collapsed;
		mediaPlayerElement.PointerMoved += MediaPlayerElement_PointerMoved;
	}

Reply:
Yes I think that is a much better approach.